apolloschurchapp/ios/Podfile

```diff
@@ -20,31 +24,21 @@ target 'apolloschurchapp' do
   # Enables Flipper.
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
-  # you should disable these next few lines.
+  # you should disable the next line.
   use_flipper!('Flipper' => '0.75.1', 'Flipper-Folly' => '2.5.3', 'Flipper-RSocket' => '1.3.1')
```

Comment:
I think we can omit the "custom" flipper versions now, also. that was a patch for our previous version of RN

Reply:
it's still listed in a thread I found somewhere as a problem with 64 too